### ISSUE TITLE
Duplicate entry for RGBA hexadecimal notation

### ISSUE THREAD

**Issue by alejandroiglesias** [Fyrd/caniuse#5101]
There are currently two entries for RGBA hex notation:

https://caniuse.com/#feat=mdn-css_properties_color_alpha_hexadecimal_notation
https://caniuse.com/#feat=css-rrggbbaa

They have different information for browser compatibility, the second being more complete.

**Comment by Fyrd** [comment]
Yup, one's the one from the recently added MDN browser compat data, the second one's from the existing caniuse database. There are indeed some slight differences, would be good determine which is the most correct.
